### PR TITLE
Style: Put a border between pinned and unpinned tabs

### PR DESCRIPTION
**Pull request by KWierso**
This should help for issue #387

### CODE REVIEW

**Comment by KWierso** [comment]
This seemed quicker and easier than completely reorganizing the pinned tabs section to get them to just be favicons.

File: skin/light/light.css [hunk 1]
```diff
@@ -241,6 +241,10 @@
   border-left: 4px solid #0996f8 !important;
 }
 
+.tabbrowser-tab[pinned] + .tabbrowser-tab:not([pinned]) {
+  border-top: 1px solid black !important;
+}
```
Comment (bwinton) [inline skin/light/light.css]: What happens when we drag the first unpinned tab down to a different spot in the tabstrip?

Comment (KWierso) [inline skin/light/light.css]: During the drag, the border follows the tab being dragged. Once the drag is finished, the border goes to the newly first unpinned tab.

Comment (bwinton) [inline skin/light/light.css]: Yeah, the border should really stay where it is…  Also when you create a new tab, the border disappears and fades in, but it shouldn't.  :(

**Comment by bwinton** [comment]
(I'm going to close this pull request, because no-one has worked on it for a while, and it has problems I'm not happy to merge it in with…)

**Pull request closed without merging**